### ISSUE TITLE
Warning: You are using the legacy PDF interpreter

### ISSUE THREAD

**Issue by silverdr** [bitcrowd/chromic_pdf#322]
I keep getting:

```
**** Warning: You are using the legacy PDF interpreter (-dNEWPDF=false) which is now deprecated and unsupported
```
even if I don't see the switch being defined anywhere. Any clues?

ChromicPDF tag: 1.16.1
Ghostscript: 10.00.0

**Comment by maltoe** [comment]
Are you sure you're on 10.0? AFAIK they've removed the old interpreter in 10.2, hence the removal of the switch here https://github.com/bitcrowd/chromic_pdf/commit/6a45f521d4256c6c9056853493bf4e757efc1220

**Comment by silverdr** [comment]
I run Debian "bookworm" (current stable) and it seems that this is what it provides:
```
$ docker container exec e83af8e5279d gs --version
10.00.0
```

**Comment by maltoe** [comment]
@silverdr Yes, sorry, my bad, brain didn't fully work there. We've introduced the `NEWPDF` switch for GS 9.56 compatibility back then (see https://github.com/bitcrowd/chromic_pdf/issues/153). In 10.0, they introduced the warning you're seeing, and in 10.2 they removed the old interpreter altogether, so we've dropped the switch for that version and above. The particular warning you're seeing is "expected" between 10.0 and 10.2.
